.travis.yml

```diff
@@ -157,11 +158,6 @@ script:
       presto-product-tests/bin/run_on_docker.sh \
         singlenode-kerberos-hdfs-impersonation-with-wire-encryption -g storage_formats,cli,hdfs_impersonation,authorization
     fi
-  - |
-    if [[ -v PRODUCT_TESTS_SPECIFIC_ENVIRONMENT_2 ]]; then
```

Comment:
I like moving this out of `PRODUCT_TESTS_SPECIFIC_ENVIRONMENT_2`, but would prefer to have a new `KAFKA_TESTS` job.